net/interfaces/interfaces.go

```diff
@@ -456,7 +456,8 @@ func UseInterestingInterfaces(i Interface, ips []netip.Prefix) bool {
 }
 
 // UseInterestingIPs is an IPFilter that reports whether ip is an interesting IP address.
-// An IP address is interesting if it is neither a loopback nor a link local unicast IP address.
+// An IP address is interesting if it is not a loopback, link local unicast IP address,
```

Comment:
This isn't a particularly useful definition of interesting. This is saying what the code is doing (and is likely to go out of date). What would be useful here is a description of why it's being done. Why are some IP addresses interesting?

I ask, because my default assumption is "interesting" means "there might be a tailscale node reachable on this." But the problem is, an IPv6 ULA absolutely could have an interesting tailscale node on it. So I do not understand the PR.

Reply:
I don't like most of this either.

Reply:
A rewording of some of the referenced bug and some extra details: we've seen some nodes rebinding more often than we want as a result of ULA range addresses getting added & removed quite regularly, but those additions and removals are not affecting / siblings of changes to a prefix delegated address or link local address. In searching around to answer the review question whether we see ULA-only nodes with working IPv6 NAT I also turned up an example node that seemed to be following the pattern Brad saw directly, which is it had over 20(!!!) ULA addresses which were identical except for the second address byte (inside the ULA prefix).